docs/dsp-tools-excel.md

```diff
@@ -1,29 +1,50 @@
+file.
+
+## Create the resources for a data model from an Excel file
+
+With dsp-tools the `resources` section used in a data model (JSON) can be created from an Excel file. Only the first worksheet of
+the Excel file is considered and only `XLSX` files are allowed. The `resources` section can be inserted into the ontology file and
+then be uploaded onto a DSP server.
+
+The Excel sheet must have the following format:
+![img-resources-example.png](assets/images/img-resources-example.png)
+
+The expected columns are:
+
+- `name` : The name of the resource
+- `super` : The base resource of the resource
+- `en`, `de`, `fr`, `it` : The labels of the resource in different languages, at least one language has to be provided
+
+For further information about resources, see [here](./dsp-tools-create.md#resources).
+
+## Create the properties for a data model from an Excel file
+
 [not yet implemented]
 
 ## Create a DSP-conform XML file from an Excel file
+
 [not yet implemented]
 
 ## Create a list from one or several Excel files
-With dsp-tools a JSON list can be created from one or several Excel files. The list can then be inserted into a JSON ontology 
-and uploaded to a DSP server. The expected format of the Excel files is described [here](./dsp-tools-create.md#lists-from-excel). 
-It is possible to create multilingual lists. In this case, a separate Excel file has to be created for each language. The data 
-has to be in the first worksheet of the Excel file(s). It is important that all the Excel lists have the same structure. So, 
-the translation(s) of a label in one Excel sheet has to be in the exact same cell (i.e. with the same cell index) in its own 
-Excel sheet.
-
-Only Excel files with file extension `.xlsx` are considered. All Excel files have to be located in the same directory. When 
-calling the `excel` command, this folder is provided as an argument to the call. The language of the labels has to be provided in 
-the Excel file's file name after an underline and before the file extension, p.ex. `liste_de.xlsx` would be considered a list with 
-German (`de`) labels, `list_en.xlsx` a list with English (`en`) labels. The language has to be a valid ISO 639-1 or ISO
-639-2 language code.
+
+With dsp-tools a JSON list can be created from one or several Excel files. The list can then be inserted into a JSON ontology and
+uploaded to a DSP server. The expected format of the Excel files is described [here](./dsp-tools-create.md#lists-from-excel). It
+is possible to create multilingual lists. In this case, a separate Excel file has to be created for each language. The data has to
+be in the first worksheet of the Excel file(s). It is important that all the Excel lists have the same structure. So, the
+translation(s) of a label in one Excel sheet has to be in the exact same cell (i.e. with the same cell index) in its own Excel
+sheet.
```

Comment:
are those linebreak changes needed?  
they can't be hidden by github, so they clutter the diffs, without changing anything at the output.

Reply:
(this applies to multiple places, I won't comment it everywhere)